src/csharp/examples/server/API/DoCoolStuff/DoCoolStuffController.cs

```diff
@@ -0,0 +1,61 @@
+using Microsoft.AspNetCore.Mvc;
+using System;
+using SessionlessExample.Server.Models;
+using System.Text.Json;
+using SessionlessNET.Impl;
+using SessionlessNET.Models;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SessionlessExample.Server.Controllers
+
+{
```

Comment:
in .NET 6 or 7 (i forgot) you are able to simply have a file-scoped namespace 

```csharp
namespace SessionlessExample.Server.Controllers;
```
instead of 
```csharp
namespace SessionlessExample.Server.Controllers { /* ... */ }
```